tests/integration/test_package_managers.py

```diff
@@ -0,0 +1,47 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import pytest
+import os
+import utils
+import json
+from git import Repo
+
+# TODO: QUESTION - How we want to store configuration parts?
```

Comment:
Can we set it as variable with fixture? Not sure what is the right approach.

Reply:
What sort of configuration are we talking about here? The config file that's not implemented yet?

Reply:
> What sort of configuration are we talking about here? The config file that's not implemented yet?

> Does it make sense that we need some environment variable to determine the usage of a local image built on the current changes?

To answer questions from @fepas and @chmeliik 
Yes, we're currently talking about configuration, which is not implemented. I would like to have environment variable at least for passing Cachi2 image from Github CI or locally built image or at last providing default quay.io latest image. 

For inspiration e.g. osbs-integration-tests keep pytest configuration within tox.ini file.
* https://gitlab.cee.redhat.com/osbs/osbs-integration-tests/-/blob/master/tox.ini#L112

We could rewrite `CACHI2_IMAGE` env var whenever it's needed?

Does this looks reasonable to you?

> Can we set it as variable with fixture? Not sure what is the right approach.
Resolving Cachi2 image - pulling will be handled in fixture yes.

Reply:
Env. vars set in tox.ini sounds good to me :+1:

Reply:
If no one objects, the configuration will be stored in tox.ini in the pytest section as described above.